packages/fxa-auth-server/fxa-oauth-server/lib/grant.js

```diff
@@ -107,6 +110,16 @@ module.exports.validateRequestedGrant = async function validateRequestedGrant(
     }
   }
 
+  // clients with server enforced PPID rotation are not allowed to
+  // request profile information to prevent the RP or service provider
+  // from learning the user's true identity.
+  if (
```

Comment:
I just realized this won't work for the secure proxy team. Removing. The secure proxy addon still needs to request profile information so they can display user information in the addon UI. To prevent the secure proxy provider from learning the user's true identity, they are requesting two access tokens, one with the "profile" scope, the other with a scope meant to access the proxy.